src/components/buttons/button-modifiers.tokens.json

```diff
@@ -1,5 +1,23 @@
 {
   "of": {
+    "login-button-logo": {
```

Comment:
probably neater to keep components isolated and instead create a file `src/components/login-button-logo/tokens.json`